lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -135,7 +130,7 @@ OpenCLFunction::OpenCLFunction(std::unique_ptr<IRFunction> F)
   // side using integer types of the same width.
   addIntOption(options, "SIZEOF_HOST_SIZE_T", sizeof(size_t));
   // Create the program from the source.
-  createProgram(SHADER_CODE, options, commands_);
+  createProgram((const char *)SHADER_CODE, options, commands_);
```

Comment:
Can you use C++-style casts here, e.g. `static_cast`

Reply:
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2440	'static_cast': cannot convert from 'const unsigned char *' to 'const char *'	OpenCL	e:\ci-cor-ready\ai\face-io\deps\glow\glow\lib\backends\opencl\opencl.cpp	133